src/util/system.cpp

```diff
@@ -1361,7 +1361,7 @@ fs::path AbsPathForConfigVal(const fs::path& path, bool net_specific)
     if (path.is_absolute()) {
         return path;
     }
-    return fsbridge::AbsPathJoin(GetDataDir(net_specific), path);
+    return fsbridge::AbsPathJoin(net_specific ? gArgs.GetDataDirNet() : gArgs.GetDataDirBase(), path);
```

Comment:
162b12cbc1051dbb8a25f271d0def1985f7cd73e: This can be squashed into d5f1458ea3821cdf87954f012bd1606659cf818f?

Reply:
Yes, it should. I can do it if another touch is needed.

Reply:
My ACK is the only one. I am happy to re-ACK if you decide to push.